lemur/notifications/messaging.py

```diff
@@ -101,6 +101,9 @@ def send_notification(event_type, data, targets, notification):
         notification.plugin.send(event_type, data, targets, notification.options)
         status = SUCCESS_METRIC_STATUS
     except Exception as e:
+        current_app.logger.error(
```

Comment:
we have learned that structural data allows for better search and sorting, when looking at the logs, for instance via  kibana. 
This is the style we have been adopting
https://github.com/Netflix/lemur/blob/b96657994aae639c626a7501daaf3822643bde99/lemur/common/celery.py#L455-L472

Reply:
I updated the log messages in this file to match that pattern. I'm not sure if there are any particular guidelines on which attributes we structure out, but I added ones that seemed useful.